src/svelte/src/components/ServerMetrics/ServerMetrics.svelte

```diff
@@ -107,32 +107,49 @@ limitations under the License.
         on:mouseenter={() => showHeartbeatInfo(true)}
         on:mouseleave={() => showHeartbeatInfo(false)}
       >
-        {#if heartbeat.latency < 20}
+        {#if 0 < heartbeat.latency && heartbeat.latency < 20}
           <circle cx="50%" cy="50%" r="4pt" fill="green" />
-        {:else if heartbeat.latency < 40}
+        {:else if 0 < heartbeat.latency && heartbeat.latency < 40}
           <circle cx="50%" cy="50%" r="4pt" fill="yellow" />
-        {:else if heartbeat.latency > 60}
+        {:else if 0 < heartbeat.latency && heartbeat.latency > 60}
           <circle cx="50%" cy="50%" r="4pt" fill="red" />
         {:else}
           <circle cx="50%" cy="50%" r="4pt" fill="grey" />
         {/if}
       </svg>
       <div class="heartbeat-info">
-        <b>Latency:</b>
-        {heartbeat.latency}ms,
-        <b>CPU Load Avg:</b>
-        {(heartbeat.serverCpuLoadAverage
-          ? heartbeat.serverCpuLoadAverage
-          : 0
-        ).toFixed(2)},
-        <b>Memory Usage:</b>
-        {heartbeat.serverUsedMemory},
-        <b>Session Count:</b>
-        {heartbeat.sessionCount},
-        <b>Uptime:</b>
-        {prettyPrintUptime(heartbeat.serverUptime)}
-        <b>JVM Version:</b>
-        {heartbeat.jvmVersion}
+        {#if heartbeat.latency > 0}
+          <b>Latency:</b>
+          {heartbeat.latency}ms,
+        {/if}
+        {#if heartbeat.serverUptime > 0}
+          <b>Uptime:</b>
+          {prettyPrintUptime(heartbeat.serverUptime)}
+        {/if}
+        {#if heartbeat.sessionCount > 0}
+          <b>Session Count:</b>
+          {heartbeat.sessionCount},
+        {/if}
+        {#if heartbeat.serverCpuLoadAverage > 0}
```

Comment:
Because of this change, windows does not display CPU Load Avg. at all because it is not >0? 

Windows:
![image](https://github.com/apache/daffodil-vscode/assets/59569025/0fbcb2d5-59af-4369-84d8-e4c8813f1d3a)


Linux:
![image](https://github.com/apache/daffodil-vscode/assets/59569025/de2a971c-201e-4dc4-88ce-c8d86de06b49)

Reply:
Correct, you and @stricklandrbls said the same.  I wanted to avoid placeholders simply due to (lack of) real estate.

Reply:
> I wanted to avoid placeholders simply due to (lack of) real estate.

The using `humanReadableByteLength(byteLength: number): string ` from `./src/svelte/src/utilities/display.ts` on the "Memory Usage" value might help with the real estate. I'm be fine approving this PR with omitting the `CPULoadAverage` value for Windows and possibly creating another ticket for restructuring the way server information is output.